sdks/python/container/boot.go

```diff
@@ -95,6 +95,7 @@ func main() {
 	// (3) Invoke python
 
 	os.Setenv("PIPELINE_OPTIONS", options)
+	os.Setenv("SEMI_PERSISTENT_DIRECTORY", dir)
```

Comment:
<!--new_thread; commit:57de5fac722898dbd4e39eb09233a5798ffcd42a; resolved:0-->
Small comment: this is passing in the 'staged' subdirectory of the semi-persistent dir. This might cause confusion.

Also, if you later want to use the space for something else as well, you'd want to use a different part of it. If you pass in the root, you'll need to update the location where you look for the session file.

Reply:
Fixed, thank you for catching this.